### DISCUSSION TITLE
ChainNotConfiguredError: Chain not configured

### DISCUSSION THREAD

**Discussion by AndonMitev** [wevm/wagmi#3550] [General]
Hey, I'm having some issues with:

```
"viem": "^2.7.1",
"wagmi": "^2.5.5"
```

Trying to connect to base goerli/sepolia but i get all the time  - ChainNotConfiguredError: Chain not configured. 

I do have another app that is utilizing wagmi@^1 and is working as expected, any ideas?

**Comment by tmm** [comment]
No ideas unless you share a minimal reproduction.

**Comment by LYD4C** [comment]
same error, chainId: 81457. this is blast network.

**Reply by LYD4C** [reply]
https://github.com/wevm/wagmi/issues/3425

` useEffect(() => {
    const wagmiStore = localStorage.getItem('wagmi.store')
    const wagmiData = wagmiStore ? JSON.parse(wagmiStore || '') : ''
    if (wagmiData && wagmiData?.chainId !== yourprojectchain.id) {
      localStorage.removeItem('wagmi.store')
    }

  }, [])`
